lib/src/migrators/calc_interpolation.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 Google LLC
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:sass_api/sass_api.dart';
+
+import '../migration_visitor.dart';
+import '../migrator.dart';
+import '../patch.dart';
+
+/// Removes interpolation in calculation functions.
+class CalculationInterpolationMigrator extends Migrator {
+  final name = "calc-interpolation";
+  final description = r"Removes interpolation in calculation functions"
+      r"`calc()`, `clamp()`, `min()`, and `max()`";
+
+  @override
+  Map<Uri, String> migrateFile(
+      ImportCache importCache, Stylesheet stylesheet, Importer importer) {
+    var visitor = _UnaryMigrationVisitor(importCache, migrateDependencies);
+    var result = visitor.run(stylesheet, importer);
+    missingDependencies.addAll(visitor.missingDependencies);
+    return result;
+  }
+}
+
+class _UnaryMigrationVisitor extends MigrationVisitor {
```

Comment:
This name should be changed to something specific to this migrator (e.g. `_CalculationInterpolationVisitor`)